test/parallel/test-os.js

```diff
@@ -116,8 +116,8 @@ const interfaces = os.networkInterfaces();
 switch (platform) {
   case 'linux':
   {
-    const filter = (e) => e.address === '127.0.0.1';
-    const actual = interfaces.lo.filter(filter);
+    const fltr = (e) => e.address === '127.0.0.1' && e.netmask === '255.0.0.0';
```

Comment:
Not a big deal but FWIW, the `fltr` const name is now inconsistent with the `filter` used in the `win32` case